src/index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--28">
+      <div class="calendar__day calendar__day--sun"></div>
```

Comment:
this modifier u should set to calendar directly

Reply:
![image](https://user-images.githubusercontent.com/51528776/183437215-897487d2-4c7f-4314-baab-49fd32366ac9.png)
what shouold I do ?

Reply:
![image](https://user-images.githubusercontent.com/51528776/185803899-49d0138d-e8e9-4afd-aad7-35980893420b.png)